kmath-tensors/src/commonTest/kotlin/space/kscience/kmath/tensors/core/TestDoubleTensor.kt

```diff
@@ -0,0 +1,86 @@
+package space.kscience.kmath.tensors.core
+
+import space.kscience.kmath.nd.DefaultStrides
+import space.kscience.kmath.nd.MutableBufferND
+import space.kscience.kmath.nd.as1D
+import space.kscience.kmath.nd.as2D
+import space.kscience.kmath.structures.DoubleBuffer
+import space.kscience.kmath.structures.asMutableBuffer
+import space.kscience.kmath.structures.toDoubleArray
+import space.kscience.kmath.tensors.core.algebras.DoubleTensorAlgebra
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+
+class TestDoubleTensor {
```

Comment:
Test classes are usually made `internal` to prevent problems if someone imports a test source set.